robust-websocket.js

```diff
@@ -239,5 +247,16 @@
     }
   }
 
+  ['CONNECTING' , 'OPEN', 'CLOSING', 'CLOSED'].forEach(function (name) {
+    const descriptor = {
+      writable: false,
+      enumerable: true,
+      configurable: false,
+      value: WebSocket[name],
+    }
+    Object.defineProperty(RobustWebSocket, name, descriptor)
+    Object.defineProperty(RobustWebSocket.prototype, name, descriptor)
+  })
```

Comment:
I'd be glad to take this change at `robust-websocket`'s new home, [nathanboktae/robust-websocket](https://github.com/nathanboktae/robust-websocket)